src/ai/defaultai.cc

```diff
@@ -7149,7 +7165,8 @@ uint32_t DefaultAI::msites_built() const {
 void DefaultAI::print_stats(const Time& gametime) {
 
 	if (!kPrintStats) {
-		set_taskpool_task_time(Time(), SchedulerTaskId::kPrintStats);
+		set_taskpool_task_time(
+		   Time(), SchedulerTaskId::kPrintStats);  // NOCOM what does Time() returns??
```

Comment:
Regarding the nocom 
it is returning never to silence this task for the rest of the game 
see
https://github.com/widelands/widelands/blob/ac9152863fddca92f6ced92e455705d55dcd5314/src/base/times.h#L143